packages/client/src/network/socket.ts

```diff
@@ -50,6 +50,7 @@ export default class Socket {
 
             // Create a SocketIO connection with the url generated.
             this.connection = io(url, {
+                transports: ['websocket'],
```

Comment:
@Veradictus This should be checked if we can do this. It fixes some flaky tests, but I am not sure if removing polling wont have side effects?

Reply:
It shouldn't affect Kaetram since the way it uses Socket.io is primarily for compatibility rather than its more fancy features. We quite literally just use raw websockets in a way.

Reply:
Very good! Then from my side I think the PR is ready for merge after review. I'll close this thread